src/PayPal/Api/Plan/BillingCycle/PriceScheme/Price.php

```diff
@@ -0,0 +1,44 @@
+<?php
+declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Swag\PayPal\PayPal\Api\Plan\BillingCycle\PriceScheme;
+
+use Swag\PayPal\PayPal\Api\Common\PayPalStruct;
+
+class Price extends PayPalStruct
```

Comment:
could also extend from https://github.com/shopwareLabs/SwagPayPal/blob/master/src/PayPal/Api/Common/Value.php

Reply:
this file is obsolete...:D i will remove it , see FixedPrice :)